test/unit/models/offsite_link_test.rb

```diff
@@ -39,14 +39,28 @@ class OffsiteLinkTest < ActiveSupport::TestCase
   test 'should be valid with whitelisted urls' do
     whitelisted_urls = [
       'http://www.flu-lab-net.eu',
-      'http://www.tse-lab-net.eu'
+      'http://www.tse-lab-net.eu',
+      'http://beisgovuk.citizenspace.com',
+      'http://www.nhs.uk',
     ]
     whitelisted_urls.each do |url|
       assert build(:offsite_link, url: url).valid?, "#{url} not valid"
     end
   end
 
-  test 'should be valid if the type is not supported' do
+  test 'should not be valid with almost whitelisted urls' do
+    whitelisted_urls = [
+      'http://someotherflu-lab-net.eu',
+      'http://a.n.othertse-lab-net.eu',
+      'http://almostbeisgovuk.citizenspace.com',
+      'http://notthenhs.uk',
```

Comment:
That test on staging was to manually find urls that might match.  I'll check again with the new code when it's on integration.